docs/using-pre-commit.md

```diff
@@ -27,35 +27,41 @@ Note that these hooks do not update the `.ipynb` notebook when you pull. Make su
 
 ## Using Jupytext with the pre-commit package manager
 
-Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file:
-```
+Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file to convert all staged notebooks to python scripts in `py:percent` format (the default):
+
+```yaml
 repos:
--   repo: local
+-   repo: https://github.com/mwouts/jupytext
+    rev: master
```

Comment:
This really should be pinned to a specific version or left for the user to specify. Otherwise it can cause a bunch of issues.

Reply:
Again, this is an example.